### DISCUSSION TITLE
Reber in Revit model do not display

### DISCUSSION THREAD

**Discussion by Simplefair** [iTwin/itwinjs-core#1497] [General]
Sir，

I try to convert a Rvit building with Snapshot (Revit 1.1.0.11). Some rebar of concrete of structural slab can be display, but all rebar of concrete structural  column can't display. I can find the element in the hierarchy, but I pick it up on hierarchy, imodel viewer display nothing. I try to input this model into Navisworks and convert it again with Snapshot (Navsiworks 1.0.0.60), the viewer can display everything correctly. The revit file is big then 50MB. If it is possible, please give me a mail address to upload it to you . 

Thanks very much!

**Comment by jffmarker** [comment]
We have reached out to our Revit library vendor to see how we can better support rebar. Revit doesn't store rebar geometry in their files most of the time, so geometry for them generally does not come across.

**Comment by Simplefair** [comment]
Thank for your quick response. I hope it can be fixed soon.

**Reply by jffmarker** [reply]
Our Revit Connector team says that RVT files do not store geometry for rebar elements. We would have to find a way to extract the rebar parameters from the RVT file and generate our own geometry to match. Product Management is currently evaluating whether to support this feature. I'd recommend raising the idea on https://itwinplatform.ideas.aha.io/ideas/ so that we know how many users are affected by this to prioritize it.

**Reply by Simplefair** [reply]
Get it and thank you very much!
